src/main.rs

```diff
@@ -108,6 +108,10 @@ enum Commands {
         /// Number of results, upto a maximum of 1000.
         #[arg(short = 'n', long)]
         limit: Option<usize>,
+
+        /// Disable automatic following of pagination responses, which can make multiple RPC calls.
+        #[arg(long, default_value_t = false)]
+        no_auto_paginate: bool,
     },
```

Comment:
i think pagination can be opt in?

Reply:
opt-out seems like a little less of a foot gun? also seems to be what AWS does: https://docs.aws.amazon.com/cli/v1/userguide/cli-usage-pagination.html#cli-usage-pagination-nopaginate

did you have something in mind why it might be better to opt in?

Reply:
ah didnt know aws does like this!